.cmake/pyre_tests.cmake

```
@@ -54,7 +54,7 @@ function(pyre_test_testcase_shell_fixture setup cleanup testfile)
     )
   # specify the required working directory
   set_property(TEST ${testname}.cleanup PROPERTY
-    WORKING_DIRECTORY ${PYRE_TESTSUITE_DIR}/${dir}
+    WORKING_DIRECTORY ${CMAKE_SOURCE_DIR}/${dir}
```

```suggestion
    WORKING_DIRECTORY ${PROJECT_SOURCE_DIR}/${dir}
```

CMAKE_SOURCE_DIR will be the wrong directory if pyre is used as a subproject via `add_subdirectory`. I think PROJECT_SOURCE_DIR is preferable for all of these.